tasks/status.rb

```diff
@@ -0,0 +1,30 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+# frozen_string_literal: true
+
+require 'open3'
+require 'puppet'
+
+begin
+  output, status = Open3.capture2('choco', 'list', '--local-only', '--no-color')
+  raise Puppet::Error, output if status != 0
+
+  result = []
+
+  output.split("\n").each do |line|
+    parts = line.split(' ')
+
+    next if parts.count != 2
+    next if parts[0] == 'Chocolatey' # package name is "chocolatey", "Chocolatey" is the header line
+
+    result << {
+      package: parts[0],
+      version: parts[1],
+    }
+  end
+
+  puts result.to_json
+  exit 0
+rescue Puppet::Error => e
+  puts({ status: 'failure', error: e.message }.to_json)
+  exit 1
+end
```

Comment:
@neomilium suggests looking into the puppetlabs/ruby_task_helper module for convenient helpers.

Reply:
Done